diff -uNr sope-mod_ngobjweb.orig/GNUmakefile sope-mod_ngobjweb/GNUmakefile
--- sope-mod_ngobjweb.orig/GNUmakefile	2006-01-23 11:02:45.000000000 +0100
+++ sope-mod_ngobjweb/GNUmakefile	2006-01-23 18:23:45.000000000 +0100
@@ -78,7 +78,8 @@
 SHARED_LIBEXT = .so
 
 CFLAGS  = -Wall -I. -fPIC \
-	$(APXS_CFLAGS) $(APXS_INCLUDE_DIRS) $(APR_INCLUDE_DIRS)
+	$(APXS_CFLAGS) $(APXS_INCLUDE_DIRS) $(APR_INCLUDE_DIRS) \
+	-D_LARGEFILE64_SOURCE -I. -I/usr/include/apr -I/usr/include/apr-util -I/usr/include/apache
 
 LDFLAGS = $(APXS_LDFLAGS) -shared -fPIC
 
