--- sope-mod_ngobjweb/GNUmakefile.old	2005-08-08 06:15:29.000000000 +0200
+++ sope-mod_ngobjweb/GNUmakefile	2005-09-05 00:22:05.000000000 +0200
@@ -56,7 +56,7 @@
 LD            = gcc
 SHARED_LIBEXT = .so
 
-CFLAGS  = -Wall -I. -fPIC $(APXS_CFLAGS) $(APXS_INCLUDE_DIRS)
+CFLAGS  = -Wall -D_LARGEFILE64_SOURCE -I. -I/usr/include/apr -I/usr/include/apr-util -I/usr/include/apache -fPIC $(APXS_CFLAGS) $(APXS_INCLUDE_DIRS)
 LDFLAGS = $(APXS_LDFLAGS) -shared -fPIC
 
 # adaptor
